### DISCUSSION TITLE
Regarding the table of contents, can the table of contents only be displayed at the top of the document?

### DISCUSSION THREAD

**Discussion by fredeason** [gsantner/markor#1768] [Ideas] [answered]
I think the current directory/outline display is very awkward.

If I have a document with a lot of levels, for example, I have read the middle of the document now, and then want to switch to a certain directory for reading, then I have to slide to the top of the document and click on the directory to jump to the corresponding place, this process It is a little troublesome for the sliding process of long documents. Can you use the method of triggering the directory, such as using a floating ball, click the harness directory, or get a button on the title bar and click to display the directory

**Comment by gsantner** [comment, marked as answer]
Document view mode or edit mode? Different things.

**Reply by fredeason** [reply]
In view mode, if you want to switch the directory, you have to slide to the top of the document to switch the directory. If the document is very long, this is very inconvenient. I wonder if the document directory can be evoked in another way, for example, adding a suspension button, click to show the directory

Don't you have this trouble?

**Reply by gsantner** [reply]
You can long press the title and it jumps to the top.

**Comment by guanglinn** [comment]
Like this?

![Screenshot_20230622_071940_net.gsantner.markor_test.jpg](https://github.com/gsantner/markor/assets/60415467/03304ec5-0e56-4661-ba3e-b4dd31675f98)
